MASFoundation/Classes/MAS.m

```diff
@@ -623,6 +635,17 @@ + (void)startWithURL:(NSURL *)url completion:(MASCompletionErrorBlock)completion
             
             NSURLSessionAuthChallengeDisposition disposition = NSURLSessionAuthChallengePerformDefaultHandling;
             
+            //
+            // Bypass SSL Pinning if its disabled.
+            //
+            if (![MAS isSSLPinningEnabled]) {
```

Comment:
Should we reset the security config over MASConfiguration?
`
 [MASConfiguration setSecurityConfiguration:securityConfig error:nil];`

Reply:
Here, we are in the SDK start method, by this time we have not kicked in the SDK services.
And the MASSecurityPolicy is extracting the MASSecurityConfiguration per host basis. So we need to overwrite here.